Backpack/Alert/Classes/BPKAlertController.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <Backpack/Color.h>
```

Comment:
These can all be moved to the `.m` file and forward declared where required

Reply:
I moved them but I could not forward declare the protocol with 
`@protocol BPKAlertViewDelegate;` So I had to leave the BPKAlertView.h import in the header.

Reply:
Yeah protocols and superclasses are fine to import in header files